Duo_S/NuttX/README_zh.md

```diff
@@ -92,7 +90,9 @@ cd nuttx
 tools/configure.sh milkv_duos:nsh
 make -j$(nproc)
 ```
-
+构建过程中如果遇到文件丢失，可以通过menuconfig配置选项
+- 在(Top) → Library Routines → Select math library 选择是否使用数学库，及其路径
+- 在(Top) → Library Routines → Builtin libclang_rt.profile选择是否启用llvm
```

Comment:
I suggest adding spaces between English and Chinese words to make them look tidy.
Not necessary though, it's up to you to decide.
Same applies to all reports.